recipes/default.rb

```diff
@@ -17,13 +17,13 @@
 # limitations under the License.
 #
 
-if node['splunk']['disabled']
+if node['splunk']['disabled'] == true
```

Comment:
Is there a reason for changing these? Since the default is `false` just `if node['splunk']['disabled']` will work.

Reply:
I've been burned by ruby before where checking `if node['splunk']['disabled']` could return a false positive. This could also return `true` if the attribute merely exists. The value could be null, false, or true. I've come into common practice of explicitly checking for a value that I know it should be or needs to be.

Reply:
```
irb(main):001:0> x = 'true'
=> "true"
irb(main):002:0> if x
irb(main):003:1> puts 'it was true'
irb(main):004:1> else
irb(main):005:1> puts 'it is not true'
irb(main):006:1> end
it was true
```